packages/backend/src/panels/ExploreGensPanel.ts

```diff
@@ -25,19 +21,14 @@ export class ExploreGensPanel extends AbstractWebviewPanel {
     this.viewTitle = "Explore and Install Generators";
     this.focusedKey = "exploreGens.Focused";
     this.htmlFileName = path.join("exploregens", "index.html");
-    this.exploreGens = new ExploreGens(
-      this.logger,
-      this.isInBAS,
-      this.context,
-      vscode
-    );
+    this.exploreGens = new ExploreGens(this.logger, this.isInBAS, this.context);
   }
 
-  public loadWebviewPanel() {
+  public async loadWebviewPanel() {
     if (this.webViewPanel) {
       this.webViewPanel.reveal();
     } else {
-      super.loadWebviewPanel();
+      return super.loadWebviewPanel();
```

Comment:
why else return value but if is not ?

Reply:
super.loadWebviewPanel requires returning a promise<void>